deeppavlov/models/kbqa/tree_to_sparql.py

```diff
@@ -169,7 +386,8 @@ class TreeToSparql(Component):
         Class for building of sparql query template using syntax parser
     """
 
-    def __init__(self, sparql_queries_filename: str, lang: str = "rus", adj_to_noun: RuAdjToNoun = None, **kwargs):
+    def __init__(self, sparql_queries_filename: str, syntax_parser: Component, kb_prefixes: Dict[str, str],
+                 lang: str = "ru", adj_to_noun: RuAdjToNoun = None, **kwargs):
```

Comment:
English version does not need syntax parser, because it uses classification model to define sparql query template, in Russian language there are not enough data for training the classification model.